### PR TITLE
#134209517 send game invite

### PR DESCRIPTION
**Pull request by tomily1**
#### What does this PR do?
This PR searches for users from the database and send an invites to registered and unregistered users.
#### Description of Task to be completed?
Search for users from the database and send invites with the URL for them to join the game. The invites can be sent to both registered and unregistered users.
#### How should this be manually tested?
The result of the test can be seen by running the test and viewing the result of the api routes created.
#### Any background context you want to provide?
No background context
#### What are the relevant pivotal tracker stories?
Users should be able to send invites to at most 11 people other than themselves
#### Screenshots (if appropriate)
screen shot 2016-12-05 at 6 34 24 pm)
#### Questions:
No questions.

### CODE REVIEW

File: app/controllers/api/invites.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**
+ * Module dependencies
+ */
+/* eslint amd:true */
+
+const nodemailer = require('nodemailer');
+require('dotenv').config({ silent: true });
+
+exports.emailinvite = (req, res) => {
```
Comment (andela-jare) [inline app/controllers/api/invites.js:9]: Don't use abbreviation

File: app/controllers/api/search.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+exports.users = (req, res) => {
+  if (req.user && req.user._id) {
+    const query = req.params.email || '';
+    User.find({ email: { $regex: query } }).limit(10)
+      .exec((err, result) => {
+        if (err) {
+          return res.json(err);
+        } else if (!result) {
+          // hope this never happens.
+          return res.send('I did not find any data');
```
Comment (ngozi-ekekwe) [inline app/controllers/api/search.js:22]: return a status code along with the response

Comment (vonvick) [reply app/controllers/api/search.js:22]: will work on it.

**Comment by tomily1** [comment]
This PR is closed because package.json is conflicting with the staging as result of mongoose.

**Pull request closed without merging**